book/chapter-12/README.md

```diff
@@ -105,13 +107,13 @@ class Tags extends React.Component {
   ...
 ```
 
-The code above together with `shouldComponentUpdate` leads to React rendering the `<ul>` with two items and then *tag-it* transforms it to a working tag editing widget.
+ตัวโค้ดที่อยู่ด้านบนกับเมธอด `shouldComponentUpdate` นำไปสู่การที่ React เรนเดอร์ตัว `<ul>` กับสองไอเท็ม และจากนั้น *tag-it* จะทำการแปลงมันเพื่อทำการแก้ไข tag widget
```

Comment:
ผมลอง paraphase "นำไปสู่การที่ React เรนเดอร์ตัว `<ul>` กับสองไอเท็ม และจากนั้น *tag-it* จะทำการแปลงมันเพื่อทำการแก้ไข tag widget" ดู ได้เป็นแบบนี้ครับ

```
ทำให้ React จะ render ตัว `<ul>`  ที่มีสองอัน หลังจากนั้นตัว *tag-it* จะแปลงมันให้เป็น widget สำหรับการแก้ไขตัว tag
```